redBullFetch.py

```diff
@@ -39,8 +43,9 @@ def generateScreenCaps(avFile, fps=3,
             raise OSError('Need an empty directory to generate screencaps!')
 
     # Generate screencaptures using ffmpeg
+    # Reference: https://ffmpeg.org/ffmpeg-filters.html#fps
     subprocess.run(['ffmpeg', '-i', './' + avFile, '-vf', 'fps=' + str(fps),
-        screenDir + '/img%05d.bmp'])
+        screenDir + '/img%06d.'+outType.lower()])
```

Comment:
The only reason would be oversampling. If the tool is always used properly, then five digits is enough. But in an oversampling condition the namespace would deplete and FFMPEG would continuously overwrite the last filename. Adding another digit just adds a little wiggle room and doesn't cost anything.